From kamalesh@linux.vnet.ibm.com  Mon Aug 17 12:39:36 2009
From: Kamalesh Babulal <kamalesh@linux.vnet.ibm.com>
Date: Fri, 14 Aug 2009 15:52:50 +0530
Subject: Staging: sep: SEP driver build breaks with CONFIG_PCI=n
To: Greg KH <gregkh@suse.de>
Cc: <mark.a.allyn@intel.com>
Message-ID: <20090814102250.GA4162@linux.vnet.ibm.com>
Content-Disposition: inline


next-20090813 randconfig build breaks Discretix SEP driver when
configured with CONFIG_PCI=n.

drivers/staging/sep/sep_driver.c: In function 'sep_probe':
drivers/staging/sep/sep_driver.c:2548: error: implicit declaration of function 'pci_dev_get'

This patch adds the dependency on PCI for the DX SEP driver.

Signed-off-by: Kamalesh Babulal <kamalesh@linux.vnet.ibm.com>
Cc: Alan Cox <alan@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/sep/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/sep/Kconfig
+++ b/drivers/staging/sep/Kconfig
@@ -1,7 +1,7 @@
 config DX_SEP
 	tristate "Discretix SEP driver"
 #	depends on MRST
-	depends on RAR_REGISTER
+	depends on RAR_REGISTER && PCI
 	default y
 	help
 	  Discretix SEP driver
